types/lhr/audit-details.d.ts

```diff
@@ -26,6 +27,23 @@ type Details =
 
 // Details namespace.
 declare module Details {
+  type NetworkNode = {
+    [id: string]: {
+      url: string;
+      navStartToEndTime: number;
+      transferSize: number;
+      children?: NetworkNode;
+    }
+  };
+
+  interface NetworkTree extends BaseDetails {
+    type: 'network-tree';
```

Comment:
This new detail type is introduced to address the differences in how the network tree is displayed in RPP vs the old CRC audit:
- The CRC audit will show individual request durations but the new NDT insight will show the entire chain duration
- The CRC audit will only show any timing/transfer size info for leaf node requests, but the NDT insight will show this info for all requests
- The CRC audit contains extra request timing info like `responseReceivedTime` but we don't need this for the NDT insight.

See `report/renderer/crc-details-renderer.js` for more info on how these detail types are handled.